data/mods/Tanks/items.json

```diff
@@ -351,7 +351,7 @@
   "recoil": 15,
   "durability": 8,
   "burst": 5,
-  "clip_size": 5,
+  "clip_size": 20,
```

Comment:
Ideally you should try mimic the ammo belts for the other Turrets for this weapon